app/src/main/java/com/example/myapplication/MatchData.java

```diff
@@ -89,56 +107,87 @@ public void setSpeakerAuto(int speakerAuto) {
     public boolean isBroke() {
         return broke;
     }
+
     public void setBroke(boolean broke) {
         this.broke = broke;
     }
 
     public boolean isDefense() {
         return defense;
     }
+
     public void setDefense(boolean defense) {
         this.defense = defense;
     }
 
     public int getAmpTeleop() {
         return ampTeleop;
     }
+
     public void setAmpTeleop(int ampTeleop) {
         this.ampTeleop = ampTeleop;
     }
 
     public int getMissedNotes() {
         return missedNotes;
     }
+
     public void setMissedNotes(int missedNotes) {   this.missedNotes = missedNotes;   }
 
```

Comment:
Fix spacing here?

Reply:
nice catch, that's the inconsistency issue that was floating about